src/Autocomplete/assets/src/controller.ts

```diff
@@ -46,18 +46,21 @@ export default class extends Controller {
     #getCommonConfig(): Partial<TomSettings> {
         const plugins: any = {}
 
-        // multiple values excepted if this is NOT A select (i.e. input) or a multiple select
-        const isMultiple = !this.selectElement || this.selectElement.multiple;
-        if (!this.formElement.disabled && !isMultiple) {
-            plugins.clear_button = { title: '' };
-        }
-
-        if (isMultiple) {
-            plugins.remove_button = { title: '' };
-        }
-
-        if (this.urlValue) {
-            plugins.virtual_scroll = {};
+        // automaticly add plugins when NOT explicit configured
+        if (!this.tomSelectOptionsValue || !this.tomSelectOptionsValue.plugins !== undefined) {
```

Comment:
Is this line right? That extra `!` in front of `this.tomSelectOptionsValue.plugins` looks funny to me.

I'd love to have a test for this. It would live in https://github.com/symfony/ux/blob/2.x/src/Autocomplete/assets/test/controller.test.ts. In the existing `pre-connect` listener - https://github.com/symfony/ux/blob/2.x/src/Autocomplete/assets/test/controller.test.ts#L23 - we could add:

```js
this.element.tomSelectOptions = event.detail.options;
```

Then, we could add a test that passes a `data-autocomplete-tom-select-options-value="{ plugins: { virtual_scroll: {} } }"` and verify with something like `getByTestId(container, 'main-element').tomSelectOptions.plugins.toBe({ virtual_scroll: {} })`.